server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -1454,7 +1456,7 @@ private RouterHealthCheckResultVO parseHealthCheckVOFromJson(final long routerId
     }
 
     private GetRouterMonitorResultsAnswer fetchAndUpdateRouterHealthChecks(DomainRouterVO router, boolean performFreshChecks) {
-        if (!RouterHealthChecksEnabled.valueIn(router.getDataCenterId())) {
+        if (!RouterHealthChecksEnabled.value()) {
```

Comment:
the value was changed to scope zone, so why not ask for valueIn(Zone)?

Reply:
The scope is changed to Global for this config Daan.